ci-scripts/yocto.groovy

```diff
@@ -46,7 +46,7 @@ void repoInit(String manifest, String yoctoDir) {
     }
 }
 
-void setupBitbake(String yoctoDir, String templateConf, boolean archive, boolean smokeTests=false) {
+void setupBitbake(String yoctoDir, String templateConf, boolean archive, boolean smokeTests=false, boolean analyzeImage=false) {
```

Comment:
I just want to make sure that this works, since it has two optionals. I believe we have had issues with this before in groovy.